### ISSUE TITLE
Deep Reinforcement Learning-Based Mapless Crowd Navigation with Perceived Risk of the Moving Crowd for Mobile Robots. (arXiv:2304.03593v2 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#27819]
Deep Reinforcement Learning-Based Mapless Crowd Navigation with Perceived Risk of the Moving Crowd for Mobile Robots. (arXiv:2304.03593v2 [cs.RO] UPDATED)<br>
https://ift.tt/5XwBTY1<br>
<p>Current state-of-the-art crowd navigation approaches are mainly deep
reinforcement learning (DRL)-based. However, DRL-based methods suffer from the
issues of generalization and scalability. To overcome these challenges, we
propose a method that includes a Collision Probability (CP) in the observation
space to give the robot a sense of the level of danger of the moving crowd to
help the robot navigate safely through crowds with unseen behaviors. We studied
the effects of changing the number of moving obstacles to pay attention during
navigation. During training, we generated local waypoints to increase the
reward density and improve the learning efficiency of the system. Our approach
was developed using deep reinforcement learning (DRL) and trained using the
Gazebo simulator in a non-cooperative crowd environment with obstacles moving
at randomized speeds and directions. We then evaluated our model on four
different crowd-behavior scenarios. The results show that our method achieved a
100% success rate in all test settings. We compared our approach with a current
state-of-the-art DRL-based approach, and our approach has performed
significantly better, especially in terms of social safety. Importantly, our
method can navigate in different crowd behaviors and requires no fine-tuning
after being trained once. We further demonstrated the crowd navigation
capability of our model in real-world tests.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
